app/views/pages/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<body>
+
+  <header>
+    <% highlight = Work.highlight %>
+    <% unless !highlight %>
+      <h2>Media Spotlight: <%= link_to "#{highlight.title}", work_path(highlight.id) %> by <%= highlight.creator %></h2>
+      <p>
+        <%= highlight.vote_count %> votes - <%= highlight.description %>
+      </p>
+    <% end %>
+  </header>
+
+  <main>
+
+    <% @categories.each do |category|%>
+      <% top_list = Work.top_works(category) %>
+      <% unless top_list == [] %>
+        
```

Comment:
I'm not a huge fan of calling model methods directly from the view like this. Would it be possible to "tee up" this data in the controller?